ethcore/src/block.rs

```diff
@@ -423,18 +400,13 @@ impl LockedBlock {
 		self.block.header.set_receipts_root(
 			ordered_trie_root(self.block.receipts.iter().map(|r| r.rlp_bytes()))
 		);
-		// compute hash and cache it.
-		self.block.header.compute_hash();
```

Comment:
Assume this was redundant

Reply:
yes, it was!

Reply:
Seems like it will be computed when actually needed (in `seal` or `try_seal` below).